packages/devtools_app/lib/src/theme.dart

```diff
@@ -226,36 +223,53 @@ extension DevToolsColorScheme on ColorScheme {
       isLight ? Colors.grey[300] : Colors.grey[700];
 
   // Title of the hover card.
-  TextStyle get hoverTitleTextStyle => const TextStyle(
-        color: Colors.black,
-        fontWeight: FontWeight.bold,
-        fontSize: 16,
+  TextStyle get hoverTitleTextStyle => TextStyle(
+        color: isLight ? Colors.black87 : Colors.white54,
+        fontWeight: FontWeight.normal,
+        fontSize: 15,
         decoration: TextDecoration.none,
       );
 
-  // Items in the hover vard.
-  TextStyle get hoverTextStyle => const TextStyle(
-        color: Colors.black,
-        fontWeight: FontWeight.bold,
+  // Items in the hover card.
+  TextStyle get hoverTextStyle => TextStyle(
+        color: isLight ? Colors.black87 : Colors.white54,
+        fontWeight: FontWeight.normal,
         fontSize: 11.5,
         decoration: TextDecoration.none,
       );
 
   // Value of items in hover e.g., capacity, etc.
-  TextStyle get hoverValueTextStyle => const TextStyle(
-        color: Colors.black,
+  TextStyle get hoverValueTextStyle => TextStyle(
+        color: isLight ? Colors.black87 : Colors.white30,
         fontWeight: FontWeight.normal,
         fontSize: 11.5,
         decoration: TextDecoration.none,
       );
 
   // Used for custom extension event values.
-  TextStyle get hoverSmallValueTextStyle => const TextStyle(
-        color: Colors.black,
+  TextStyle get hoverSmallValueTextStyle => TextStyle(
+        color: isLight ? Colors.black87 : Colors.white30,
         fontWeight: FontWeight.normal,
         fontSize: 10,
         decoration: TextDecoration.none,
       );
+
+  // Used for ExpansionTile collapsed trailing icon color.
+  Color get unselectedColor => isLight ? Colors.black : Colors.white;
```

Comment:
can we use contrastForeground instead where you are currently using unselectedColor?

Reply:
Yep done and removed the unselectedColor theme